README.md

```diff
@@ -39,3 +39,4 @@ KAFKA_PRODUCER_RETRIES | 5 |
 LOG_LEVEL | DEBUG |
 MAX_RETRY_ATTEMPTS | 10 | Number of retries before the message is added to the error topic.
 RETRY_THROTTLE_RATE_SECONDS | 30 | Delay between retry consumer processing the messages on the retry topic.
+
```

Comment:
For future you can just use an empty commit
`git commit --allow-empty -m "Trigger Build"`